test/e2e/scenarios/multiple_clusters.go

```diff
@@ -0,0 +1,14 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("Namespace selector based Network Policies for ingress between clusters, selecting multiple remote clusters", func() {
+	PIt("Should allow communication between pods in selected namespace across multiple clusters", func() {
+		By("creating pod1 with label1 in cluster1 in namespace1")
+		By("creating pod2 with label2 in cluster2 in namespace1")
+		By("creating network policy with namespace selector in more than 2 clusters")
+		By("testing connectivity between pods")
```

Comment:
we could create another pod, without the right label, and make sure it fails to connect.